### DISCUSSION TITLE
Accessing Firefox with VNC - accetto/ubuntu-vnc-xfce-firefox-g3

### DISCUSSION THREAD

**Discussion by r4zi3lrrr** [accetto/ubuntu-vnc-xfce-g3#66] [General]
Hi,

Does anyone know if there is a way to save my Firefox session e.g. home page, favourites etc whenever I go back into Firefox VNC?

If I log out of VNC Firefox, I have to set everything backup again.

I tried creating a default profile and bookmarking a website, but it always resets to its original setup.

Also accetto/ubuntu-vnc-xfce-firefox-g3 keeps on closing up on me using both Ubuntu and Debian, seems to auto-close on it's own accord.

**Comment by markNZed** [comment]
Sounds like the docker session might be unstable. If docker container is restarting then you will lose local setup unless you bind a volume with the data stored outside the docker container. If you share your docker setup then someone might give a better answer.
